### PR TITLE
add readme

### PR DESCRIPTION
**Pull request by AxelVivien25**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# mlflow-elasticsearchstore
+
+Mlflow plugin to use Elaticsearch backend in the Tracking Store of Mlflow. To use this plugin you need a running instance of Elasticsearch 6.X.
+
+## Development
+
+In a python environment :
```
Comment (jcuquemelle) [inline README.md:7]: "in the same python env where you installed mlflow" ?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+```bash
+$ git clone git clone https://github.com/criteo/mlflow-elasticsearchstore.git
+$ cd mlflow-elasticsearch
+$ pip install .
+```
+
+## How To 
+
+mlflow-elasticsearchstore can now be used with the entrypoint elasticsearch, in the same python environment :
```
Comment (jcuquemelle) [inline README.md:17]: (...) used with the "elasticsearch" scheme (...)

File: README.md (hunk 1 above)
Comment (AxelVivien25) [reply README.md:7]: Not needed, mlflow is reinstalled with the requirements.txt if needed

File: README.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# mlflow-elasticsearchstore
+
+Mlflow plugin to use Elaticsearch backend in the Tracking Store of Mlflow. To use this plugin you need a running instance of Elasticsearch 6.X.
```
Comment (t-henri) [inline README.md:3]: I'd say 'use ElasticSearch as backend for MLflow tracking service.'

**Pull request merged**